### ISSUE TITLE
[v2] Cannot run app

### ISSUE THREAD

**Issue by natebrennand** [adicu/Courses#180]
After following the main route of steps I encountered this error:

```
➜  Courses git:(backend_meteor) ✗ meteor
[[[[[ ~/github/Courses ]]]]]

=> Started proxy.
=> Started MongoDB.
=> Starting your app...
/Users/nate/.meteorite/packages/bower/mquandalle/meteor-bower/5699c3f29a936dcb2aa1221a9febd4669764fc91/.build/plugin.bower.os/npm/bower/plugin/node_modules/bower/node_modules/q/q.js:126
                    throw e;
                          ^
Error: Package authors not found
  at createError (/Users/nate/.meteorite/packages/bower/mquandalle/meteor-bower/5699c3f29a936dcb2aa1221a9febd4669764fc91/.build/plugin.bower.os/npm/bower/plugin/node_modules/bower/lib/util/createError.js:4:15)
  at /Users/nate/.meteorite/packages/bower/mquandalle/meteor-bower/5699c3f29a936dcb2aa1221a9febd4669764fc91/.build/plugin.bower.os/npm/bower/plugin/node_modules/bower/lib/core/resolverFactory.js:143:27
  at _fulfilled (/Users/nate/.meteorite/packages/bower/mquandalle/meteor-bower/5699c3f29a936dcb2aa1221a9febd4669764fc91/.build/plugin.bower.os/npm/bower/plugin/node_modules/bower/node_modules/q/q.js:787:54)
  at self.promiseDispatch.done (/Users/nate/.meteorite/packages/bower/mquandalle/meteor-bower/5699c3f29a936dcb2aa1221a9febd4669764fc91/.build/plugin.bower.os/npm/bower/plugin/node_modules/bower/node_modules/q/q.js:816:30)
  at Promise.promise.promiseDispatch (/Users/nate/.meteorite/packages/bower/mquandalle/meteor-bower/5699c3f29a936dcb2aa1221a9febd4669764fc91/.build/plugin.bower.os/npm/bower/plugin/node_modules/bower/node_modules/q/q.js:749:13)
  at /Users/nate/.meteorite/packages/bower/mquandalle/meteor-bower/5699c3f29a936dcb2aa1221a9febd4669764fc91/.build/plugin.bower.os/npm/bower/plugin/node_modules/bower/node_modules/q/q.js:557:44
  at flush (/Users/nate/.meteorite/packages/bower/mquandalle/meteor-bower/5699c3f29a936dcb2aa1221a9febd4669764fc91/.build/plugin.bower.os/npm/bower/plugin/node_modules/bower/node_modules/q/q.js:108:17)
  at process._tickCallback (node.js:419:13)
```

Also could not get Docker to work but that wasn't as well documented

```
➜  Courses git:(backend_meteor) ✗ docker build -t courses .
Sending build context to Docker daemon
2014/07/24 10:33:02 Post http:///var/run/docker.sock/build?rm=1&t=courses: dial unix /var/run/docker.sock: no such file or directory
```

**Comment by brishin** [comment]
Yeah, docker isn't supported right now. Let me actually just get rid of that package (it's a wrapper around bower, but what I was using it for is now a meteor package as well now)

**Comment by brishin** [comment]
Should be closed in d8fe7391d75b427144ef125be8226ba93ff38e82

**Comment by natebrennand** [comment]
Thanks, I'll try it again this evening
